### PR TITLE
lab5-var1 Елагин Денис 6213

### PR DESCRIPTION
**Pull request by Donistr**
добавил датасет и блокнот с 5 лабой.
ссылка на коллаб: https://colab.research.google.com/drive/1JpyTj7rDjO5cCYXLXFa3Vcot3X8VJtSd#scrollTo=sz-euS_F-yBJ

### CODE REVIEW

File: Lab4/Untitled2.ipynb [hunk 1]
```diff
@@ -0,0 +1,1759 @@
...
...00:00\",\"2007-07-05T00:00:00\",\"2007-07-06T00:00:00\",\"2007-07-07T00:00:00\",\"2007-07-10T00:00:00\",\"2007-07-11T00:00:00\",\"2007-07-12T00:00:00\",\"2007-07-13T00:00:00\",\"2007-07-14T00:00:00\",\"2007-07-17T00:00:00\",\"2007-07-18T00:00:00\",\"2007-07-19T00:00:00\",\"2007-07-20T00:00:00\",\"2007-07-21T00:00:00\",\"2007-07-24T00:00:00\",\"2007-07-25T00:00:00\",\"2007-07-26T00:00:00\",\"2007-07-27T00:00:00\",\"2007-07-28T00:00:00\",\"2007-07-31T00:00:00\",\"2007-08-01T00:00:00\",\"2007-08-02T00:00:00\",\"2007-08-03T00:00:00\",\"2007-08-04T00:00:00\",\"2007-08-07T00:00:00\",\"2007-08-08T00:00:00\",\"2007-08-09T00:00:00\",\"2007-08-10T00:00:00\",\"2007-08-11T00:00:00\",\"2007-08-14T00:00:00\",\"2007-08-15T00:00:00\",\"2007-08-16T00:00:00\",\"2007-08-17T00:00:00\",\"2007-08-18T00:00:00\",\"2007-08-21T00:00:00\",\"2007-08-22T00:00:00\",\"2007-08-23T00:00:00\",\"2007-08-24T00:00:00\",\"2007-08-25T00:00:00\",\"2007-08-28T00:00:00\",\"2007-08-29T00:00:00\",\"2007-08-30T00:00:00\",\"2007-08-31T00:00:00\",\"2007-09-01T00:00:00\",\"2007-09-04T00:00:00\",\"2007-09-05T00:00:00\",\"2007-09-06T00:00:00\",\"2007-09-07T00:00:00\",\"2007-09-08T00:00:00\",\"2007-09-11T00:00:00\",\"2007-09-12T00:
```
Comment (alxmcs) [inline Lab4/Untitled2.ipynb:1326]: о, что validation loss = 0 совершенно никак тебя не напрягло?

**Review by alxmcs — CHANGES_REQUESTED** [review]
Судя по всему, ты и #59 смотрели в один и тот же туториал и допустили один и тот же косяк

**Review by alxmcs — APPROVED** [review]
